lib/plugins/inspec-habitat/README.md

```diff
@@ -0,0 +1,149 @@
+# InSpec Habitat Plugin
+
+## Summary
+
+This plugin allows you to do the following:
+  1. Add Habitat configuration to a profile
+  2. Create/Upload a Habitat package from an InSpec profile
+
+Creating a [Habitat](https://www.habitat.sh/) package from an InSpec profile
+allows you to execute that profile as a service (via a Habitat Supervisor) on
+any Linux based platform.
+
+When running as a service, an InSpec report will be created in JSON format (by
+default at `/hab/svc/YOUR_SERVICE/logs/inspec_last_run.json`).  Additionally, a
+log of the last run can be located at
+`/hab/svc/YOUR_SERVICE/logs/inspec_last_run.json` and CLI output is viewable in
+the Supervisor logs by default. You can also configure this service to report
+to [Chef Automate](https://www.chef.io/automate/).
+
+See below for usage instructions.
+
+## Plugin Usage
+
+### Adding Habitat Configuration to an InSpec Profile
+
+Run the following command:
+
+```
+inspec habitat profile setup PATH
+```
+
+This will create the following files:
+  - habitat/plan.sh (Instructs Habitat on how to build the package)
```

Comment:
Ahh, @kagarmoe 	will know best, but "on how" reads colloqiual to me.  Rephrase?

Reply:
I'll change to `Provides build time instructions to Habitat`. Thoughts?